src/routes/package-edit.js

```diff
@@ -117,6 +128,11 @@ class PackageEditRoute extends Component {
         model.contentType = values.contentType;
       }
 
+      if ('proxyId' in values) {
+        model.proxy.id = values.proxyId;
+        model.proxy.inherited = false;
+      }
+
```

Comment:
Really minor thing in packageEditSubmitted function -- may want to reset `model.proxy = {}` when package is being de-selected

Reply:
Addressed 👍